youtube/templates/base.html

```diff
@@ -4,7 +4,10 @@
         <meta charset="utf-8">
         <title>{{ page_title }}</title>
         <meta http-equiv="Content-Security-Policy" content="default-src 'self' 'unsafe-inline'; media-src 'self' https://*.googlevideo.com;
-            {{ "img-src https://*.googleusercontent.com https://*.ggpht.com https://*.ytimg.com;" if not settings.proxy_images else "" }}">
+            {% if not settings.proxy_images %}
+                img-src https://*.googleusercontent.com https://*.ggpht.com https://*.ytimg.com;
+            {% endif %}
+
```

Comment:
This broke the html. Should be removed. The second thing I recommend prior to making commits is to test all functionality related to or impacted by the code that was changed (for instance, if you make a change in the code for comments, test every piece of commenting functionality affected by the change; video comments, reply page, more comments page, etc.). This is the second way to avoid committing bugs. It's tedious, but will catch a lot of bugs you did not anticipate when performed religiously.